src/utils/isAxiosError.ts

```diff
@@ -1,6 +1,6 @@
-import { isAxiosError } from '../../node_modules/axios/index.js';
+import { isAxiosError } from 'axios';
 
 // Our dependency on Axios is indirect through Zodios.
-// Zodios doesn't re-export isAxiosError, so we need to import it haphazardly through node_modules.
+// Zodios doesn't re-export isAxiosError, so we need to import it directly from axios.
```

Comment:
Please revert the changes to this file. The comment explains why it imports from node_modules directly.